examples/examples_go_test.go

```diff
@@ -251,12 +251,16 @@ func (st tagsState) validateStateResult(phase int) func(
 			t.Logf("key=%s tags are as expected: %v", k, actualTagsJSON)
 
 			if k == "bucket" {
+				// getTags := stack.Outputs["get-bucket"].(string)
+				// assert.Equal(t, v.(string), getTags)
 				bucketName := stack.Outputs["bucket-name"].(string)
 				st.assertTagsEqualWithRetry(t,
 					fetchBucketTags(bucketName),
 					"bad bucket tags")
 			}
 			if k == "legacy-bucket" {
+				// getTags := stack.Outputs["get-legacy-bucket"].(string)
```

Comment:
Are these comments still useful?

Reply:
Added another comment with link to the issue. When the issue is fixed the code can be uncommented.